Doc/whatsnew/3.13.rst

```diff
@@ -650,583 +654,669 @@ ast
-  and :meth:`~pathlib.Path.rglob`.
+* Add *recurse_symlinks* keyword-only argument to
+  :meth:`.Path.glob` and :meth:`~pathlib.Path.rglob`.
   (Contributed by Barney Gale in :gh:`77609`.)
 
-* Add *follow_symlinks* keyword-only argument to :meth:`~pathlib.Path.is_file`,
-  :meth:`~pathlib.Path.is_dir`, :meth:`~pathlib.Path.owner`,
-  :meth:`~pathlib.Path.group`.
-  (Contributed by Barney Gale in :gh:`105793`, and Kamil Turek in
-  :gh:`107962`.)
-
-* Return files and directories from :meth:`pathlib.Path.glob` and
-  :meth:`~pathlib.Path.rglob` when given a pattern that ends with "``**``". In
-  earlier versions, only directories were returned.
+* :meth:`.Path.glob` and :meth:`~pathlib.Path.rglob`
+  now return files and directories when given a pattern that ends with "``**``".
+  Previously, only directories were returned.
   (Contributed by Barney Gale in :gh:`70303`.)
 
-pdb
----
+* Add the *follow_symlinks* keyword-only argument to
+  :meth:`Path.is_file <pathlib.Path.is_file>`,
+  :meth:`Path.is_dir <pathlib.Path.is_dir>`,
+  :meth:`.Path.owner`, and :meth:`.Path.group`.
+  (Contributed by Barney Gale in :gh:`105793` and Kamil Turek in :gh:`107962`.)
 
-* Add ability to move between chained exceptions during post mortem debugging in :func:`~pdb.pm` using
-  the new ``exceptions [exc_number]`` command for Pdb. (Contributed by Matthias
-  Bussonnier in :gh:`106676`.)
 
-* Expressions/statements whose prefix is a pdb command are now correctly
-  identified and executed.
-  (Contributed by Tian Gao in :gh:`108464`.)
+:mod:`pdb`
+----------
 
-* ``sys.path[0]`` will no longer be replaced by the directory of the script
-  being debugged when ``sys.flags.safe_path`` is set (via the :option:`-P`
-  command line option or :envvar:`PYTHONSAFEPATH` environment variable).
+* :func:`breakpoint` and :func:`~pdb.set_trace` now enter the debugger immediately
+  rather than on the next line of code to be executed. This change prevents the
+  debugger from breaking outside of the context when :func:`!breakpoint` is positioned
+  at the end of the context.
+  (Contributed by Tian Gao in :gh:`118579`.)
+
+* ``sys.path[0]`` is no longer replaced by the directory of the script
+  being debugged when :attr:`sys.flags.safe_path` is set.
   (Contributed by Tian Gao and Christian Walther in :gh:`111762`.)
 
-* :mod:`zipapp` is supported as a debugging target.
+* :mod:`zipapp` is now supported as a debugging target.
   (Contributed by Tian Gao in :gh:`118501`.)
 
-* ``breakpoint()`` and ``pdb.set_trace()`` now enter the debugger immediately
-  rather than on the next line of code to be executed. This change prevents the
-  debugger from breaking outside of the context when ``breakpoint()`` is positioned
-  at the end of the context.
-  (Contributed by Tian Gao in :gh:`118579`.)
+* Add ability to move between chained exceptions during
+  post mortem debugging in :func:`~pdb.pm` using
+  the new ``exceptions [exc_number]`` command for Pdb.
```

Comment:
Is this "Pdb" the class `Pdb` or module `pdb` or something else?

Reply:
I think it means the Pdb command-line, I've added a cross-reference which might help things, or we could add clarificatory words.